app/src/Routes.jsx

```diff
@@ -33,7 +32,7 @@ export class Routes extends React.Component {
     const { user } = this.state;
     return (
       <Switch>
-        <Route path="/admin" render={props => <AdminContainer user={user} {...props} />} />
+        <Route path="/admin" render={props => <div user={user} {...props} />} />
```

Comment:
`user` isn't a valid property on div.
If the intention is to remove the non-existing admin page just remove the route too.

Reply:
oops